catt/controllers.py

```diff
@@ -110,8 +116,11 @@ def setup_cast(device_name, video_url=None, prep=None, controller=None):
 
```

Comment:
We need to fail here, if the type of the chosen cast device is not in ```supported_device_types```.

Reply:
Instead of failing, I decided to just print a warning for two reasons:

- this is what `catt` currently do for other stuff
- warnings are more future proof than failures